release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1415,6 +1415,11 @@ In the following tables, features are marked with the following statuses:
 ==== The Shared Resource CSI Driver is removed
 The Shared Resource CSI Driver feature was deprecated in {product-title} 4.17, and is now removed from {product-title} 4.18. This feature is now generally available in Builds for Red Hat OpenShift 1.1. To use this feature, ensure you are using Builds for Red Hat OpenShift 1.1 or later.
 
+[id="ocp-4-18-selected-bundled-removed_{context}"]
+==== The selected bundles feature has been removed in oc-mirror v2
+
+The selected bundles feature has been removed in oc-mirror v2 general availability. This change prevents issues where specifying the wrong operator bundle version could break operators on a cluster. Customers using the oc-mirror v2 technology preview will no longer find this feature in the GA release. (link:https://issues.redhat.com/browse/OCPBUGS-49419[*OCPBUGS-49419*])
```

Comment:
```suggestion
The selected bundles feature is removed from the oc-mirror v2 Generally Available release. This change prevents issues where specifying the wrong Operator bundle version could break the Operators in a cluster. (link:https://issues.redhat.com/browse/OCPBUGS-49419[*OCPBUGS-49419*])
```
If we aren't supporting this feature, but want folks to update to GA, I think just saying you can't do this anymore is enough.